logstash-core/src/main/java/org/logstash/Logstash.java

```diff
@@ -44,7 +44,7 @@ public static void main(final String... args) {
         ) {
             logstash.run();
         } catch (final Throwable t) {
-            LOGGER.error(t);
+            LOGGER.error(t.toString());
```

Comment:
I believe this provides a more useful message. In my case, `LOGGER.error(t)` was printing only `java.lang.ExceptionInInitializerError` while `LOGGER.error(t.toString())` printed the exception class name along with the specific JRuby class that was failing to initialize.